components/Page/PageHeader.jsx

```diff
@@ -1,10 +1,25 @@
+'use client'
+import { IconButton } from '../ui/IconButton/IconButton'
 import { Typography } from '../ui/Typography'
+import { useRouter } from 'next/navigation'
+import { Left } from '../ui/Icons'
 import clsx from 'clsx'
 
-const PageHeader = ({ title, className }) => {
+const PageHeader = ({ title, className, backpageButton = false }) => {
+  const router = useRouter()
   return (
-    <header className={clsx(className, 'mb-8')}>
-      <Typography variant="title">{title}</Typography>
+    <header className={clsx(className, 'mb-8 inline-flex')}>
+      {backpageButton == true && (
```

Comment:
You can simplify this expression with : 
```
backpageButton ? (...) : null
```
I prefer using ternary expression because && in javascript can render odd things ([source](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx))